py/torch_tensorrt/dynamo/lowering/passes/fuse_distributed_ops.py

```diff
@@ -12,16 +12,23 @@
 logger = logging.getLogger(__name__)
 
 
-def tensorrt_fused_nccl_all_gather_op(args0, args1, args2):
+# TODO: @apbose make these actual torch custom ops, should allow aot_export
+def tensorrt_fused_nccl_all_gather_op(
```

Comment:
Ok noted in the TODOs. Just to get clarity, what is meant by allowing aot_export? What is the difference in the present behavior?

Reply:
Sonthat you dont need to use the autograd workflow and could potentially support torch.export flows